### ISSUE TITLE
Support for Angular 16

### ISSUE THREAD

**Issue by Raphiiko** [danrevah/ngx-pipes#253]
Angular 16 has released this week, however this package seems to not yet be compatible. 

Attempting to install it results in the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: REDACTED@#.#.#
npm ERR! Found: @angular/core@16.0.0
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"^16.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"14 - 15" from ngx-pipes@3.2.0
npm ERR! node_modules/ngx-pipes
npm ERR!   ngx-pipes@"^3.2.0" from the root project

```

**Comment by ThrownLemon** [comment]
+1 🙏

**Comment by danrevah** [comment]
Fixed in version v3.2.2
